extensions/positron-notebook-controllers/src/notebookSessionService.ts

```diff
@@ -252,16 +251,16 @@ export class NotebookSessionService implements vscode.Disposable {
 					session = await startingSessionPromise;
 				} catch (err) {
 					log.error(`Waiting for notebook runtime to start before shutting down failed. Reason ${err}`);
-					throw err;
+
+					// If the session failed to start, we don't need to do anything.
+					return;
 				}
+			} else {
+				// If there's no session and no starting session, we don't need to do anything.
+				return;
 			}
 		}
 
-		// Ensure that we have a session.
-		if (!session) {
-			throw new Error(`Tried to shutdown runtime for notebook without a running runtime: ${notebookUri.path}`);
```

Comment:
Should we still spit something out into a log here? Or is the idea that it's not going to be possible to get to this situation anymore?

Reply:
Yeah, I'll add a debug log